recipes/pycontracts/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "PyContracts" %}
+{% set version = "1.8.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/4e/7a/0f79370e4e3a6741396d76d1f76586c2924bed049fb38597799b72a24081/{{ name }}-{{ version }}.tar.gz"
+  sha256: e76adbd832deec28b2045a6094c5bb779a0b2cb1105a23b3efafe723e2c9937a
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python  
  number: 0
```